charts/cryostat/tests/service_test.yaml

```diff
@@ -0,0 +1,62 @@
+suite: test service.yaml
+templates:
+  - service.yaml
+
+tests:
+  - it: should create a Service with the correct settings
+    set:
+      core.service.type: ClusterIP
+      core.service.httpPort: 8080
+      authentication.openshift.enabled: false
+    asserts:
+      - hasDocuments:
+          count: 1
+      - equal:
+          path: kind
+          value: Service
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-cryostat
+      - equal:
+          path: spec.type
+          value: ClusterIP
+      - equal:
+          path: spec.ports[0].port
+          value: 8080
+      - equal:
+          path: spec.ports[0].targetPort
+          value: 4180
+      - equal:
+          path: spec.ports[0].protocol
+          value: TCP
+      - equal:
+          path: spec.ports[0].name
+          value: cryostat-http
+      - equal:
+          path: metadata.labels
+          value:
+            app.kubernetes.io/instance: RELEASE-NAME
+            app.kubernetes.io/managed-by: Helm
+            app.kubernetes.io/name: cryostat
+            app.kubernetes.io/version: "4.0.0-dev"
+            helm.sh/chart: cryostat-0.5.0-dev
```

Comment:
Hey @andrewazores, I am seeing Chart version is still `0.5.0-dev` on `main`. Should this be something `1.x`?

Reply:
Nice catch, yes, probably `1.1`? @ebaron any other idea?

Reply:
1.1 or 2.0 are fine with me. No strong feeling either way.

Reply:
I guess if we are setting `4.0.0-dev` on the operator side, we can use `2.0` here since there can be upcoming breaking changes (e.g. #115)?